admin/sql/create_tables.sql

```diff
@@ -207,6 +207,15 @@ CREATE TABLE user_relationship (
     created             TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
 );
 
+CREATE TABLE pinned_track (
```

Comment:
Let's call this pinned_recording, since we're pinning recordings and not tracks (the MB db actually has tracks, which are distinct from recordings).